pkg/kf/systemenvinjector/systemenvinjector.go

```diff
@@ -26,6 +26,7 @@ import (
 // through.
 type SystemEnvInjectorInterface interface {
 	InjectSystemEnv(app *v1alpha1.App) error
+	ComputeSystemEnv(app *v1alpha1.App) (computed []corev1.EnvVar, err error)
```

Comment:
Do we still need Inject, or can that be erased?

Reply:
I don't think so. I removed it